packages/governance/src/binaryBallotCounter.js

```diff
@@ -0,0 +1,169 @@
+// @ts-check
+
+import { assert, details as X } from '@agoric/assert';
+import { makeStore } from '@agoric/store';
+import { makePromiseKit } from '@agoric/promise-kit';
+import { Far } from '@agoric/marshal';
+
+import { ChoiceMethod, buildBallot } from './ballotBuilder';
+
+const makeWeightedBallot = (ballot, shares) => ({ ballot, shares });
+
+const makeBinaryBallot = (question, positionAName, positionBName) => {
+  const positions = [];
+  assert.typeof(question, 'string');
+  assert.typeof(positionAName, 'string');
+  assert.typeof(positionBName, 'string');
+  positions.push(positionAName, positionBName);
+
+  return buildBallot(ChoiceMethod.CHOOSE_N, question, positions, 1n);
+};
+
+const makeQuorumCounter = quorumThreshold => {
+  const check = stats => {
+    const votes = stats.results.reduce(
+      (runningTotal, { total }) => runningTotal + total,
+      0n,
+    );
+    return votes >= quorumThreshold;
+  };
+  /** @type {QuorumCounter} */
+  return Far('checker', { check });
+};
```

Comment:
Can this just be:

```suggestion
  const checkQuorum = (threshold, stats) => {
    const votes = stats.results.reduce(
      (runningTotal, { total }) => runningTotal + total,
      0n,
    );
    return votes >= threshold;
  };
```

Reply:
Not objecting, but I would be surprised if the quorum checker was remote. That seems more like a module at most rather than a separate remote contract

Reply:
It's plausible; I'm moving in that direction with my implementation of closure rules, which is next. I'm just not positive that we can foresee every quorum rule, which means it has to be an external contract/component.